src/dashboard/src/main/management/commands/rebuild_transfer_backlog.py

```diff
@@ -166,22 +188,70 @@ def populate_data(self, es_client, transfer_backlog_dir):
                 self.info(
                     "Importing self-describing transfer {}.".format(transfer_uuid)
                 )
-                size = get_bag_size(bag, str(transfer_dir))
+                size = am.get_bag_size(bag, str(transfer_dir))
                 _import_self_describing_transfer(
                     self, es_client, self.stdout, transfer_dir, transfer_uuid, size
                 )
             else:
                 self.info("Rebuilding known transfer {}.".format(transfer_uuid))
                 if bag:
-                    size = get_bag_size(bag, str(transfer_dir))
+                    size = am.get_bag_size(bag, str(transfer_dir))
                 else:
-                    size = walk_dir(str(transfer_dir))
+                    size = am.walk_dir(str(transfer_dir))
                 _import_pipeline_dependant_transfer(
                     self, es_client, self.stdout, transfer_dir, transfer_uuid, size
                 )
             processed += 1
         self.success("{} transfers indexed!".format(processed))
 
+    def populate_data_from_storage_service(self, es_client, pipeline_uuid):
+        """Populate data in search indices and/or database from files temporarily downloaded from Storage Service.
```

Comment:
This might require further discussion but I think the introduction of `black` has changed what we consider long lines. Docstrings and comments are also a bit special. and [pep8 has](https://www.python.org/dev/peps/pep-0008/#id19):

> ... it is okay to increase the line length limit up to 99 characters, provided that comments and docstrings are still wrapped at 72 characters.

Reply:
Good catch -  I didn't realize that the 72 character limit was strict for docstrings and comments. I've reworked this docstring and the others so that they wrap at 72 characters.